main/zoe/guide/contracts/auto-swap.md

```diff
@@ -28,21 +28,24 @@ methods in the API available to the user:
 3. getPrice
 4. makeOffer
 
-We can call `addLiquidity` with an escrow receipt from Zoe that proves
```

Comment:
Not sure how to improve the language here, but `addLiquidity` is one of the methods on the seat, but with a seat sounds like passing an argument

Reply:
How about this:
"If we want to prove that we've escrowed moola and simoleans appropriately, we can call `addLiquidity` on the seat we receive when we redeem the invite."

Reply:
I think we should leave out the "prove" part because that was about the escrow receipt. Maybe something like, `we can contribute to the autoswap liquidity pool by calling `addLiquidity` on a seat.`